.changeset/eighty-pans-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+Force logout live clients on E2EE keys reset
```

Comment:
Should this be a minor? Isn't it more like a patch?

btw, can you explain a bit more your fix in here? 🤗

Reply:
Done!